src/main/java/com/iexec/sms/tee/session/gramine/sps/SpsConfiguration.java

```diff
@@ -38,7 +38,7 @@ public class SpsConfiguration {
     private String webPassword;
 
     @Value("${gramine.sps.enclave.host}")
```

Comment:
```suggestion
    @Value("${gramine.sps.enclave.hostname}")
```
host = hostname + port
:)

Reply:
(`host-name` or `host.name` in yml if staying consistent with `enclaveHostName` in Java but I find them weird)

Reply:
`hostname` seems better, I agree. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/116/commits/780fbbe61261a4e449fa37bb820b0f049afadd80.